primitives/TypographyWrapper.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { ReactNode } from 'react';
```

Comment:
```suggestion
import React, { type ReactNode } from 'react';
```

When importing types, let's take the practice of prepending the `type` keyword to it, so to give cue of what it is.